src/bin/cli.ts

```diff
@@ -20,7 +21,8 @@ program
   .option('--responseSuffix <name>', 'pass response suffix')
   .option('--baseUrl <url>', 'pass baseUrl')
   .option('-h, --hooks', 'generate React Hooks')
-  .option('--file <filename>', 'output file name (ex: generated.api.ts)')
+  .option('-c, --config <path>', 'pass tsconfig path for resolve path alias')
+  .requiredOption('-f, --file <filename>', 'output file name (ex: generated.api.ts)')
```

Comment:
I'm not sure if we want to make this required.

Reply:
I Changed "file" option back to "optional"

Reply:
Thanks!